src/components/navigation/problem-tab-content.tsx

```diff
@@ -36,6 +36,10 @@ export const ProblemTabContent: React.FC<IProps>
   const problemsPanelHeight = vh - headerOffset;
   const problemsPanelStyle = { height: problemsPanelHeight };
 
+  if (ui.showChatPanel) {
+    ui.updateFocusDocument();
+  }
```

Comment:
In response to the question you asked on slack about how to determine when to update the focus document, I'm thinking something like this:
```suggestion
  useEffect(() ={
    if (ui.activeNavTab === ENavTab.kProblems) {
      ui.updateFocusDocument();
    }
  }, []);
```
When we mount the component, if the active nav tab is the Problem curriculum, then update the focus document. Otherwise, leave it to the tab-switching calls.